### PR TITLE
Updates to the documentation

### PR DESCRIPTION
**Pull request by noobiept**
Hello, here are just some small updates to the documentation.

Basically to make it more in line with how the types are in typescript (HTMLImageElement instead of Image for example). You can read the commit message for more details. 
Only thing I'm not sure is about the rest parameters, don't know if that is how you write it in a docstring. 

Cheers!

### CODE REVIEW

**Comment by noobiept** [comment]
I reverted the array changes (as was determined here: https://github.com/CreateJS/PreloadJS/pull/108#discussion_r22477813).

File: src/easeljs/display/Container.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ this.createjs = this.createjs||{};
 	 * 		container.addChild(bitmapInstance, shapeInstance, textInstance);
 	 *
 	 * @method addChild
-	 * @param {DisplayObject} child The display object to add.
+	 * @param {DisplayObject|...DisplayObject} child The display object to add.
```
Comment (lannymcnie) [inline src/easeljs/display/Container.js]: I recommend removing these. They aren't parsed by YUIDocs.

Comment (noobiept) [inline src/easeljs/display/Container.js]: Yea I figured it wouldn't support as well. Updated.

Comment (MannyC) [inline src/easeljs/display/Container.js]: Looks like it should be 
`@param {DisplayObject} child* The display object to add.`
or if you update the grunt-contrib-yuidoc package then the new syntax is
`@param {DisplayObject} ...child The display object to add.`

**Pull request merged**